src/Dialogs/Message.hpp

```diff
@@ -66,4 +66,7 @@ enum {
 int
 ShowMessageBox(const TCHAR *text, const TCHAR *caption, unsigned flags);
 
+void
+ShowLockBox();
```

Comment:
Why is this declared in `Message.hpp`? This isn't a message dialog. There's no message, no text, no information. Just waiting for the user to close it.